beacon_node/beacon_chain/tests/store_tests.rs

```diff
@@ -1191,6 +1191,271 @@ fn check_shuffling_compatible(
     }
 }
 
+/// These tests check the consistency of:
+///
+/// - ProtoBlock::proposer_shuffling_root_for_child_block, and
+/// - BeaconState::proposer_shuffling_decision_root{_at_epoch}
+async fn proposer_shuffling_root_consistency_test(parent_slot: u64, child_slot: u64) {
```

Comment:
a little strange that these tests are under `store_tests.rs`, i guess it's more for convenience?